cdk_framework/EKS/lib/app.ts

```diff
@@ -1,12 +1,13 @@
 #!/usr/bin/env node
 import 'source-map-support/register';
 import * as cdk from 'aws-cdk-lib';
-import { ClusterStack } from './stacks/cluster-stack';
 import { deployClusters } from './cluster-deployment';
+import { FargateStack } from './stacks/fargate-cluster-stack';
+import { EC2Stack } from './stacks/ec2-cluster-stack';
 
 const app = new cdk.App();
 
-let clusterMap = new Map<string, ClusterStack>()
+let clusterMap = new Map<string, FargateStack | EC2Stack>()
```

Comment:
`Map<string, FargateStack | EC2Stack>()` gets reused a lot. Can we use a type alias instead of repeating this everywhere? Find a good place to define
```
type clusterMapType = Map<string, FargateStack | EC2Stack>
```